pyaedt/desktop.py

```diff
@@ -515,9 +515,8 @@ def is_student_version(oDesktop):
 
 def _init_desktop_from_design(*args, **kwargs):
     """Used to distinguish if the Desktop is initialized internally from Design or directly from the outside"""
```

Comment:
```suggestion
    """Used to distinguish if AEDT is initialized internally from Design or directly from the outside"""
```
What is "from Design or directly from outside"? Maybe "Used to distinguish if AEDT is initialized internally from the Design app or externally through PyAEDT." I feel that the current docstring is too vague.